app/controllers/campaigns_controller.rb

```diff
@@ -92,6 +94,25 @@ def set_user
 
   # Never trust parameters from the scary internet, only allow the white list through.
   def campaign_params
-    params.fetch(:campaign, {})
+    params.require(:campaign).permit(
+      :user_id,
+      :status,
+      :name,
+      :date_range,
+      :url,
+      :creative_id,
+      :us_hours_only,
+      :weekdays_only,
+      :ecpm,
+      :daily_budget,
+      :total_budget,
+      countries: [],
+      keywords: [],
+      negative_keywords: []
+    )
+  end
+
+  def sortable_columns
+    %w[name end_date total_budget_cents status created_at user.first_name]
```

Comment:
Would prefer to see this split over multiple lines and sorted. Much easier to scan.